### ISSUE TITLE
See Different Channels in Audio Source?

### ISSUE THREAD

**Issue by jdcarls2** [ChurchApps/FreeShow#210]
Finally got to test FreeShow in our production setting, and found that while I could see our audio input from our sound board, I couldn't hear anything.

Our board outputs 32 channels, and the designated output for the computer, where FreeShow is running, is pretty far down the list. When FreeShow sees a multi-channel source right now, what does it do? Is it just grabbing the first channel? Or all of them?

I suspect this is what [this note in Mic.svelte](https://github.com/vassbo/freeshow/blob/e8c13d1f2a39c56c82bb637e7cd04df46265c600/src/frontend/components/drawer/live/Mic.svelte#L12) is about? Not sure, though.

**Comment by vassbo** [comment]
Did you unmute the input?

I guess it should list all the individual channels, but can't remember.
I'll see what I can do about this.

**Comment by jdcarls2** [comment]
Yes, I unmuted the input. The built-in mic was picking up sound, so I could see audio levels coming through on that, but the external audio input had nothing.

**Comment by vassbo** [comment]
Okay.

**Comment by vassbo** [comment]
All audio inputs are added seperately. I don't have a sound board to test, but if it doesn't work in the latest version (0.9.3), then it might not be supported by Chromium? Could you try out this app with the audio board, and send me the results!?: https://brianpeiris.github.io/mic-test/

**Comment by jdcarls2** [comment]
I will try and test it this week!

**Comment by jdcarls2** [comment]
I was off AV last week, but am getting to test it today. I still see the audio source in FreeShow just fine:

<img width="1466" alt="image" src="https://github.com/vassbo/freeshow/assets/39384515/5de1d2c6-fdef-4f02-adb8-88172b75e6c4">

But I can't get the audio coming through it. I can confirm that it is working, as ProPresenter sees it and all its channels.

I went to the site you linked, and I see this:

<img width="734" alt="image" src="https://github.com/vassbo/freeshow/assets/39384515/aedc1180-e4b6-49f5-9654-20d0736ab4df">

I found another mic test, and saw this:

<img width="413" alt="image" src="https://github.com/vassbo/freeshow/assets/39384515/c7019420-72ce-4457-8cc5-08ab329e5cef">

It's only picking up a single channel! I hooked the sound board up to a Windows machine and got FreeShow running on there, and it only saw two, even in ProPresenter. There's clearly something odd going on with the Mac. I don't know if it's worth digging into too much. We're eventually going to replace our Mac with a PC that has a dedicated line in.
